src/index.js

```diff
@@ -1,36 +1,147 @@
-const tasks = [
-  {
-    description: 'Wash the dishes',
-    completed: false,
-    index: 1,
-  },
-  {
-    description: 'Completed TO Do list project',
-    completed: true,
-    index: 2,
-  },
-];
-
-document.addEventListener('DOMContentLoaded', () => {
-  const todoList = document.getElementById('todo-list');
-  todoList.innerHTML = '';
-  todoList.style.listStyle = 'none';
-
-  tasks.sort((a, b) => a.index - b.index);
-
-  tasks.forEach((task) => {
-    const li = document.createElement('li');
-    li.style.border = '1px solid #ddd';
-    li.style.display = 'block';
-    li.style.padding = '10px 0px 10px 50px';
-    li.style.marginLeft = '-40px';
-    li.style.marginRight = '50vw';
+import './style.css';
+
+const taskInput = document.getElementById('input');
+const addButton = document.getElementById('button');
+const todoList = document.getElementById('todo-list');
+const completeButton = document.querySelector('.complete');
+
+const savedTasks = JSON.parse(localStorage.getItem('tasks')) || [];
+
+/* eslint-disable */
```

Comment:
- [ ] @SAZZAD1Q2 as previously requested kindly ensure you solve the linter errors instead of disabling them.